### PR TITLE
Add instructions for installing from public apt repos

### PR DESCRIPTION
**Pull request by rbuysse**
Signed-off-by: Ryan Beck-Buysse <rbuysse@bitwise.io>

### CODE REVIEW

File: docs/source/sysadmin_guide/installation.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  These instructions have been tested on Ubuntu 16.04 only.
+
+Stable Builds
+-------------
+
+Add the Sawtooth Lake repository to your ``/etc/apt/sources.list``:
+
+.. code-block:: console
+
+  $ sudo echo "deb http://repo.sawtooth.me/ubuntu/0.8 xenial universe" >> /etc/apt/sources.list
```
Comment (aludvik) [inline docs/source/sysadmin_guide/installation.rst]: `ubuntu/0.8` => `ubuntu/0.8/stable`

File: docs/source/sysadmin_guide/installation.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+  $ apt-get install -y -q \
+    python3-sawtooth-cli \
+    python3-sawtooth-config \
+    python3-sawtooth-intkey \
+    python3-sawtooth-manage \
+    python3-sawtooth-poet-common \
+    python3-sawtooth-poet-core \
+    python3-sawtooth-poet-families \
+    python3-sawtooth-poet-simulator \
```
Comment (aludvik) [inline docs/source/sysadmin_guide/installation.rst]: + python3-sawtooth-poet-cli

Assuming you got this from the `sawtooth-all` Dockerfile, that is probably out of date now too.

File: docs/source/sysadmin_guide/installation.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    python3-sawtooth-validator \
+    python3-sawtooth-xo
+
+
+Nightly Builds
+--------------
+
+If you'd like the latest and greatest version of Sawtooth Lake, we also
+provide a repository of nightly builds. These builds may incorporate
+undocumented features and should be used for testing purposes only.
```
Comment (aludvik) [inline docs/source/sysadmin_guide/installation.rst]: I'd recommend not duplicating all the instructions again here. Maybe just reference the instruction above that would have to change, or make it a choice above.

File: docs/source/sysadmin_guide/installation.rst [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+.. code-block:: console
+
+  $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 6B58B1AC10FB5F63
+
+Update your repositories and install Sawtooth Lake:
+
+.. code-block:: console
+
+  $ sudo apt-get update
+  $ apt-get install -y -q \
```
Comment (aludvik) [inline docs/source/sysadmin_guide/installation.rst]: sudo apt-get install

File: docs/source/sysadmin_guide/installation.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+Update your repositories and install Sawtooth Lake:
+
+.. code-block:: console
+
+  $ sudo apt-get update
+
+.. code-block:: console
+
+  $ apt-get install -y -q \
```
Comment (aludvik) [inline docs/source/sysadmin_guide/installation.rst]: sudo apt-get install

File: docs/source/sysadmin_guide/installation.rst (hunk 3 above)
Comment (rbuysse) [inline docs/source/sysadmin_guide/installation.rst]: I decided to have separate instructions for nightly in case we ever rename/add/remove packages.

Comment (aludvik) [inline docs/source/sysadmin_guide/installation.rst]: That makes sense.

**Comment by boydjohnson** [comment]
This looks fine. Something that may be nice would be a dependency graph for our deb packages. And a general statement about what each package gives you.

**Pull request closed without merging**